duktape/src/main/java/com/squareup/duktape/Duktape.java

```diff
@@ -69,20 +69,22 @@ public String evaluate(String script) {
    * defines the interface implemented by {@code object} that will be accessible to JavaScript.
    * {@code type} must be an interface that does not extend any other interfaces, and cannot define
    * any overloaded methods.
+   *
+   * Methods of the interface may return void or any of the following supported argument types:
```

Comment:
You need a `<p>` before this or Javadoc will collapse the sentence into the previous paragraph.